x-pack/plugins/cases/common/constants.ts

```diff
@@ -157,3 +157,9 @@ export const READ_CASES_CAPABILITY = 'read_cases' as const;
 export const UPDATE_CASES_CAPABILITY = 'update_cases' as const;
 export const DELETE_CASES_CAPABILITY = 'delete_cases' as const;
 export const PUSH_CASES_CAPABILITY = 'push_cases' as const;
+
+/**
+ * User profiles
+ */
+
+export const DEFAULT_USER_SIZE = 100;
```

Comment:
What do you think about defaulting it to 10? I believe that's what the Elasticsearch default is. 100 seems like it could result in a pretty long list.

Reply:
Ok! Sounds good to me.